shims/spark301/src/main/scala/org/apache/spark/sql/rapids/shims/spark300/ShuffleManagerShim.scala

```diff
@@ -17,7 +17,8 @@ package org.apache.spark.sql.rapids.shims.spark301
 
```

Comment:
Happened to notice this file is in the spark301 package and shim but in a spark300 directory?!  Can clean this up in a separate PR.